scriptmodules/supplementary/splashscreen/asplashscreen.sh

```diff
@@ -10,6 +10,13 @@ do_start () {
     local config="/etc/splashscreen.list"
     local line
     local re="$REGEX_VIDEO\|$REGEX_IMAGE"
+    local splash_vt=2
+
+    # switching VT in KMS mode will break graphics acceleration on RPI4
+    if grep -q okay /proc/device-tree/soc/v3d@7ec00000/status 2> /dev/null || grep -q okay /proc/device-tree/soc/firmwarekms@7e600000/status 2> /dev/null; then
```

Comment:
it's guaranteed that `v3d@7ec00000` is stable across devices/setups?
There isn't a more robust way to check this?

Reply:
It's stable across devices. There may be a more optimal way to detect it, but I'd prefer to leave it this was as it's the precise method that raspi-config uses: https://github.com/RPi-Distro/raspi-config/blob/master/raspi-config#L74-L80

Reply:
Ah! if `raspi-config` uses it like that, then I def agree is good to leave it as-is. Thanks for clarifying!